src/components/navigation/navigation.tsx

```diff
@@ -1,14 +1,17 @@
 import { ReactNode } from "react";
 import { Route } from "next";
+import Image from "next/image";
 import Link from "next/link";
 
 import { AiOutlineHeart, AiOutlineSearch, AiOutlineShoppingCart } from "react-icons/ai";
 
 export const Navigation = () => {
   return (
-    <nav className="fixed bottom-0 z-50 w-full bg-mauve-1 sm:sticky sm:w-auto sm:self-start">
-      <div className="hidden sm:block  sm:px-3 sm:py-2">ロゴ</div>
-      <div className="flex cursor-pointer gap-1 border-t sm:flex-col sm:border-none">
+    <nav className="fixed bottom-0 z-50 w-full bg-mauve-1 sm:sticky sm:w-auto sm:self-start sm:pr-5 sm:pt-5">
+      <div className="hidden sm:mb-8 sm:block">
```

Comment:
前から気になっていたのですが、[Reactにおいてあまり`margin`は使わない方がよい](https://tech.kasuga-inc.co.jp/20210703/use-padding-instead-of-margin/)ため、以下のようにするのはどうでしょうか。
```suggestion
    <nav className="fixed bottom-0 z-50 w-full bg-mauve-1 sm:sticky sm:flex sm:w-auto sm:flex-col sm:gap-y-8 sm:self-start sm:pr-5 sm:pt-5">
      <div className="hidden sm:block">
```
オフトピですが、時間の都合上、他にも`margin`が使われている箇所があり、今まで見逃していた部分があるので、リファクタリングなどのタイミングで見直しを検討いただけますと幸いです。

Reply:
@takeyu1013 
修正しました。
[d508010](https://github.com/qin-team-recipe/08-recipe-app/pull/163/commits/d5080102b1ba4e926e9437242f0f3ece11210363)